### PR TITLE
Add oembed endpoint that returns iframe

### PR DESCRIPTION
**Pull request by andrebola**
issue #773

Follows specification on http://oembed.com/

The only required parameter is 'url' which corresponds to the sound e.g: https://freesound.org/people/Bram/sounds/11/

I also would like to specify somewhere how to use this but I'm not sure where should it go.

### CODE REVIEW

File: sounds/views.py [hunk 1]
```diff
@@ -798,6 +798,28 @@ def embed_iframe(request, sound_id, player_size):
     return render_to_response('sounds/sound_iframe.html', locals(), context_instance=RequestContext(request))
 
 
+def oembed(request):
+    url = request.GET.get('url', None)
+    view, args, kwargs = resolve(urlparse(url)[2])
```
Comment (bdejong) [inline sounds/views.py]: What happens if URL=None?

File: sounds/views.py [hunk 2]
```diff
@@ -798,6 +798,28 @@ def embed_iframe(request, sound_id, player_size):
...
+    if not 'sound_id' in kwargs:
+        raise Http404
+    sound_id = kwargs['sound_id']
+    sound = get_object_or_404(Sound, id=sound_id, moderation_state='OK', processing_state='OK')
+    player_size = request.GET.get('size', 'medium')
+    if player_size == 'large':
+        sizes = [920, 245]
```
Comment (bdejong) [inline sounds/views.py]: we should probably remove these from here and from the template and make them constants somewhere...

File: sounds/views.py (hunk 1 above)
Comment (andrebola) [inline sounds/views.py]: I changed this, now returns 404 if 'url' is not present

**Comment by bdejong** [comment]
👍

**Pull request merged**